mime-native/src/main/java/org/ballerinalang/mime/util/MimeConstants.java

```diff
@@ -34,13 +35,13 @@
     public static final String MEDIA_TYPE = "MediaType";
     public static final String CONTENT_DISPOSITION_STRUCT = "ContentDisposition";
     public static final String PROTOCOL_PACKAGE_MIME = BALLERINA_PACKAGE_PREFIX + "mime";
+    @Deprecated
     public static final String PROTOCOL_MIME_PKG_VERSION = "1.0.6";
```

Comment:
We can entirely remove this right?

Reply:
This constant has been used in HTTP also. So once the HTTP is migrated this constant will be removed. So for the moment marked as deprecated